cypress/e2e/shared/dashboardsView.test.ts

```diff
@@ -167,7 +167,7 @@ describe('Dashboard', () => {
     const headerPrefix2 = '-'
 
     cy.getByTestID('note-editor--overlay').monacoType(
-      `${headerPrefix2} ${noteText2}`
+      `\n${headerPrefix2} ${noteText2}`
```

Comment:
the `- Text` has to be on a newline, in order to trigger the markdown conversion to a list.

tests were failing due to the absence of this newline (paired with the editor version bump):
https://app.circleci.com/pipelines/github/influxdata/monitor-ci/10633/workflows/1f173ccd-f236-499e-bc1c-9e8848004d22/jobs/558131/tests#failed-test-0